_docs/desktop/adding-duckduckgo-to-your-browser.md

```diff
@@ -1,7 +1,9 @@
 ---
-title: Adding DuckDuckGo to Your Browser
-category: Desktop
-order: 9
+title: How to add DuckDuckGo to your browser
```

Comment:
We maintained an IE link on this page, but deleted the page and redirected it back here. We probably need to remove that link here.

Reply:
Good catch. I've removed the IE link.